README.md

```diff
@@ -45,9 +45,8 @@ In the meantime, Swagger has recently been added to trace-server.
 * Such reference and future (forecasted) TSP versions will be documented accordingly as these yaml files evolve.
 
 1. Import all [TC][tracecompass] and [incubator][incubator] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
-1. Run trace-server or project [1] below as Eclipse Application configured this way (or similar):
-   * `-Xms512m -Xmx1g`.
-   * Launch with `all workspace and enabled target plug-ins`.
+1. Open `traceserver.product` file in plug-in `org.eclipse.tracecompass.incubator.trace.server.product`
```

Comment:
1. Missing a trailing dot here and below, consistently with other listed items around.
2. Do we still need that reference [1] then? (Would remove it here too if not.)

Reply:
> 2\. Do we still need that reference [1] then? (Would remove it here too if not.)

Yes, we do.

Reply:
> > 2. Do we still need that reference [1] then? (Would remove it here too if not.)
> 
> Yes, we do.

We ended up not needing it anymore ([1]), and you did remove it in the sibling commit, so all fine.